src/app/models.py

```diff
@@ -53,6 +53,7 @@ class Detection(SQLModel, table=True):
     camera_id: int = Field(..., foreign_key="camera.id", nullable=False)
     azimuth: float = Field(..., gt=0, lt=360)
     bucket_key: str
+    localization: Union[str, None]
```

Comment:
cf. earlier discussion, I think we need a better name for that: keypoints or bboxes are much clearer (localization could be the actual geographic location of the wildfire or the camera)

Reply:
@MateoLostanlen